app/views/posts/index.html.erb

```diff
@@ -0,0 +1,66 @@
+<div class="desktop">
+<% @posts.in_groups_of(3) do |post_group|%>
+<div class="row">
+<% post_group.each do |post|%>
+<% if post %>
```

Comment:
Thanks for the quick fix! Just for a future reference, one of the alternatives to the fix could be to use conditional chaining. e.g. `post&.title`